### PR TITLE
Net: Turn some methods and params/variables const

### PR DESCRIPTION
**Pull request by jtimon**
Also remove currently unused interruptMsgProc param from SendMessages.
I guess some squashing can be done (but I'm worried that interruptMsgProc is planned to be used in SendMessages later).

### CODE REVIEW

File: src/net.cpp [hunk 1]
```diff
@@ -1871,7 +1871,7 @@ void CConnman::ThreadMessageHandler()
             // Send messages
             {
                 LOCK(pnode->cs_sendProcessing);
-                GetNodeSignals().SendMessages(pnode, *this, flagInterruptMsgProc);
+                GetNodeSignals().SendMessages(pnode, *this);
```
Comment (TheBlueMatt) [inline src/net.cpp]: Even if we dont use it now, I'm not sure we want to get rid of passing flagInterruptMsgProc through...there is a strong case for using it in 0.15.

**Comment by TheBlueMatt** [comment]
General concept ack, with the exception of removing the flagInterruptMsg pass.

**Comment by jtimon** [comment]
Great, it seems I was right on leaving the last commit separated, I'll leave it out.

**Review by TheBlueMatt — COMMENTED** [review]
utACK 0729102f99241b9716456ad40bf778c09f650b1b (+/- SocketSendData being made const).

File: src/net.cpp [hunk 2]
```diff
@@ -754,7 +754,7 @@ const uint256& CNetMessage::GetMessageHash() const
 
 
 // requires LOCK(cs_vSend)
-size_t CConnman::SocketSendData(CNode *pnode)
```
Comment (jtimon) [inline src/net.cpp]: Well, writing to the peer itself here is writing to pnode, which isn't const, no?

**Comment by laanwj** [comment]
utACK 0729102

**Pull request merged**